_episodes/01-numpy.md

```diff
@@ -1135,3 +1135,73 @@ the graphs will actually be squeezed together more closely.)
 > > {: .output}
 > {: .solution}
 {: .challenge}
+
+>## Change In Inflamation
+>
+>This patient data is _longitudinal_ in the sense that each row represents a
+>series of observations relating to one individual. This means that change
+>inflamation is a meaningful concept.
+>
+>The `numpy.diff()` function takes a NumPy array and returns the 
+>difference along a specified axis.
+>
+>Which axis would it make sense to use this function along?
+> > ## Solution
+> > Since the row axis (0) is patients, it does not make sense to get the
+> > difference between two arbitrary patients. The column axis (1) is in
+> > days, so the differnce is the change in inflamation -- a meaningful
+> > concept.
+> >
+> > ~~~
+> > numpy.diff(data, axis=1)
+> > ~~~
+> > {: .python}
+> {: .solution}
+>
+>If the shape of an individual data file is `(60, 40)` (60 rows and 40 columns)
+>, what would the shape of the array be after you run the `diff()` function and
+>why?
+> > ## Solution
+> > The shape will be `(60, 39)` because there is one fewer difference between
+> > columns than there are columns in the data.
+> {: .solution}
+>
+>How would you find the largest change in inflammation for each patient? Does
+>it matter if the change in inflammation is an increase or a decrease?
+> > ## Solution
+> > By using the `max()` function after you apply the `diff()` function, you
+> > will get the largest difference between days.
+> > ~~~
```

Comment:
I'm not sure if this is required by the Markdown parser either, but for clarity could add newlines before and after code blocks too? I.e.,

```
> > ... end of paragraph.
> > 
> > ~~~
> > numpy.diff(...)
> > ~~~
> > {: .python}
> >
> > Start of next paragraph...
```